cardano-db-sync/src/Cardano/DbSync/Era/Universal/Insert/Grouped.hs

```diff
@@ -61,12 +63,13 @@ data MissingMaTxOut = MissingMaTxOut
 -- reference outputs that are not inserted to the db yet.
 data ExtendedTxOut = ExtendedTxOut
   { etoTxHash :: !ByteString
-  , etoTxOut :: !DB.TxOut
+  , etoTxOut :: !DB.TxOutW
+  , etoPaymentCred :: !(Maybe ByteString)
```

Comment:
Why is this necessary? The `VTxOutW` already contains the address and the payment credential?

Reply:
I see the address in `VTxOutW` can be Nothing. But why is this allowed?

Reply:
yeah I'd done this way before doing `VTxOutW` at the time it was a way to get to `Address` payment cred. Will change 👍